### PR TITLE
Tweak pscan Alpha Help

### PR DESCRIPTION
**Pull request by kingthorin**
Remove ASP Viewstate entries.
(https://github.com/zaproxy/zap-extensions/blob/48d28560fdcfc308bc968a90abb31031c7610ae6/src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help/contents/pscanbeta.html#L76-L84)
Updated manifest.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html [hunk 1]
```diff
@@ -9,12 +9,6 @@
 <H1>Passive Scan Rules - Alpha</H1>
 The following alpha quality passive scan rules are included in this add-on:
 
-<H2>ASP.NET ViewState Disclosure</H2>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: The scanner was not promoted yet, these are raised by Base64 Disclosure.

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: Thanks, I totally missed that!

Comment (thc202) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: Does it worth put the name of the scanner after the alert name? (or add a note somewhere?)

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: I was wondering that yesterday, while reading through these.

Something like `ASP.NET ViewState Disclosure (Base64Disclosure)`

Comment (thc202) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: Looks good. I think we just need to do that in the help(?), in the other places we include the ID of the scanner or its name (e.g. Alerts tab with Source field).

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: Yup. I’ll put together a PR for Alpha shortly and we can discuss further in review of that.

Comment (thc202) [reply src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html]: Great, thanks!

**Pull request closed without merging**